contracts/CollateralFunds.sol

```diff
@@ -6,9 +6,14 @@ pragma solidity ^0.8.7;
  * @author The Everest team.
```

Comment:
Update the contract's description to something similar to this:
`A simple contract to which you can deposit or withdraw collateral funds.`